backend/app/resources/feed.py

```diff
@@ -27,6 +28,7 @@ def get(self):
         sort_vote = request.args.get("vote", None, type=str)
         sort_date = request.args.get("date", "desc", type=str)
         sort_tag = request.args.get("tag", None, type=str)
+        followed_only = request.args.get("q", None, type=str)
```

Comment:
General comment but I will just mention it here, it's best to always review the docstrings after changing a code snippet. We always need our docstrings _(which are the primary sources to interact with the api in this context)_ to be up-to-date